src/js/electron/tron/windowManager.ts

```diff
@@ -97,7 +97,7 @@ export default function windowManager() {
     openSearchTab(searchParams: NewTabSearchParams) {
       let isNewWin = true
       const existingWin = this.getWindows()
-        .sort((a, b) => a.lastFocused - b.lastFocused)
+        .sort((a, b) => b.lastFocused - a.lastFocused)
```

Comment:
this was sorted chronologically resulting in selection of first search window that had been focused on, instead of last